src/features/import/hooks/useColumn.ts

```diff
@@ -72,24 +72,28 @@ export default function useColumn(orgId: number) {
       value: `field:${fieldSlug}`,
     }));
 
-  const customFieldsOptions: Option[] = customFields.map((field) => {
-    if (field.type === CUSTOM_FIELD_TYPE.DATE) {
-      return {
-        label: field.title,
-        value: `date:${field.slug}`,
-      };
-    } else if (field.type == CUSTOM_FIELD_TYPE.ENUM && field.enum_choices) {
-      return {
-        label: field.title,
-        value: `enum:${field.slug}`,
-      };
-    } else {
-      return {
-        label: field.title,
-        value: `field:${field.slug}`,
-      };
-    }
-  });
+  const customFieldsOptions: Option[] = customFields
+    .filter(
+      (field) => field.organization.id == orgId || field.org_write == 'suborgs'
+    )
```

Comment:
Nice and simple solution! My only question is whether they should maybe still be visible, but disabled (with an extra label explaining why, e.g. "Read-only"). Either way, that's not something that needs to happen in this PR. But lets discuss with designers and potentially open another issue.